package.json

```diff
@@ -88,7 +89,10 @@
   "auto": {
     "extends": "@remusao/auto-config",
     "name": "Adblocker Bot",
-    "email": "adblocker-bot@users.noreply.github.com"
+    "email": "adblocker-bot@users.noreply.github.com",
+    "plugins": [
+      "protected-branch"
+    ]
```

Comment:
To double-check if this is merged with the list of plugins inherited from base config.